libr/core/cmd_anal.c

```diff
@@ -1488,14 +1492,10 @@ static int var_cmd(RCore *core, const char *str) {
 		}
 		if ((type == 'b') && delta > 0) {
 			isarg = true;
-		} else if (type == 's' && fcn && delta > fcn->maxstack) {
+		} else if (type == 's' && delta > fcn->maxstack) {
```

Comment:
cant add a comment above, but "name" is just used as an alias for `p`.  rename the variable.

vartype must be const char * and size is always 4, so just make it const. also add a comment right to the case with the command name that is handling there